tests/instant_articles/test_data/input/betty-no-caption.html

```diff
@@ -0,0 +1,3 @@
+<div data-type="image" contenteditable="false" class="onion-image image inline size-big crop-original" data-image-id="9513" data-size="big" data-crop="original" data-format="jpg">
+   <div></div>
```

Comment:
ah, yeah. the parser/renderer always spit out an empty caption, but i'll add it here just in case.